addon/services/scroller.js

```diff
@@ -59,7 +61,7 @@ export default Service.extend({
     let defaults = this.get('defaults');
     let axis = options.axis || defaults.axis;
     let ignoreViewport = typeof options.ignoreViewport !== 'undefined' ? options.ignoreViewport : defaults.ignoreViewport;
-    let container = options.container && this.getContainer(options.container) || window;
+    let container = options.container && this.getContainer(options.container) || this.get('window');
```

Comment:
maybe it makes sense to optionally inject/use the `window` service if available and fallback to `window` instead to avoid the additional dependency on `ember-window-mock` 🤔

Reply:
Good call. Refactoring the approach to use whatever it gets for a container, checking if `container.scrollTo` is a function to determine how we scroll, rather than comparing the container to the `window` global.